src/main/java/jalau/at18/azul/Wall.java

```diff
@@ -0,0 +1,32 @@
+package jalau.at18.azul;
+public class Wall {
+    private final int wallMatrixSize = 5;
+    private Tile[][] tiles = new Tile[wallMatrixSize][wallMatrixSize];
+    private final int two = 2;
```

Comment:
If this line is a constant you can have it like 
**`private static final int TWO = 2;`**
the same thing for lines 6 and 7 if they are constants

Reply:
Hi @RodrigoValda , not sure if there is going to be a change based on @FernandezRobert 's suggestion? otherwise, please do a rebase and we should be able to merge